components/admin-panel/sections/Team.tsx

```diff
@@ -0,0 +1,429 @@
+import React from 'react';
+import { gql, useQuery } from '@apollo/client';
+import { Edit } from '@styled-icons/material/Edit';
+import { get, truncate } from 'lodash';
+import { FormattedDate, FormattedMessage, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { FEATURES, isFeatureEnabled } from '../../../lib/allowed-features';
+import { CollectiveType } from '../../../lib/constants/collectives';
+import roles from '../../../lib/constants/roles';
+import { i18nGraphqlException } from '../../../lib/errors';
+import { API_V2_CONTEXT } from '../../../lib/graphql/helpers';
+import type {
+  Account,
+  AccountWithHost,
+  AccountWithParent,
+  Member,
+  MemberInvitation,
+} from '../../../lib/graphql/types/v2/graphql';
+import formatMemberRole from '../../../lib/i18n/member-role';
+import { getCollectivePageRoute } from '../../../lib/url-helpers';
+
+import Avatar from '../../Avatar';
+import Container from '../../Container';
+import ResendMemberInviteBtn from '../../edit-collective/ResendMemberInviteBtn';
+import EditMemberModal from '../../edit-collective/sections/EditMemberModal';
+import InviteMemberModal from '../../edit-collective/sections/InviteMemberModal';
+import { Box, Flex, Grid } from '../../Grid';
+import { getI18nLink } from '../../I18nFormatters';
+import Link from '../../Link';
+import Loading from '../../Loading';
+import MemberRoleDescription from '../../MemberRoleDescription';
+import MessageBox from '../../MessageBox';
+import StyledButton from '../../StyledButton';
+import StyledHr from '../../StyledHr';
+import StyledRoundButton from '../../StyledRoundButton';
+import StyledTag from '../../StyledTag';
+import StyledTooltip from '../../StyledTooltip';
+import { P } from '../../Text';
+import { AdminSectionProps } from '../types';
+
+const MemberContainer = styled(Container)`
+  display: block;
+  height: 100%;
+  min-height: 232px;
+  min-width: 164px;
+  background: white;
+  width: 170px;
+  border-radius: 8px;
+  border: 1px solid #c0c5cc;
+`;
+
+const TagContainer = styled(Box)`
+  position: absolute;
+  bottom: 16%;
+`;
+
+const InviteNewCard = styled(MemberContainer)`
+  border: 1px dashed #c0c5cc;
+  cursor: pointer;
+`;
+
+/** A container to center the logo above a horizontal bar */
+const MemberLogoContainer = styled(Box)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  border-top: 1px solid #e6e8eb;
+`;
+
+const memberFieldsFragment = gql`
+  fragment MemberFields on Member {
+    id
+    role
+    since
+    createdAt
+    description
+    inherited
+    account {
+      id
+      name
+      slug
+      type
+      imageUrl(height: 64)
+      ... on Individual {
+        email
+      }
+    }
+  }
+`;
+
+export const teamPageQuery = gql`
```

Comment:
`teamPageQuery` page is misleading as we're using this in the section, a better name would be `teamSectionQuery`?